frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -0,0 +1,160 @@
+
+export class RhvImportWizard extends ImportWizard {
+  async selectProvider(provider: string) {
+    await selectOptionByText(view.providerSelect, provider);
+  }
+
+  async selectInstance(instance: string) {
+    await selectOptionByText(view.vcenterInstanceSelect, instance);
+  }
+
+  async fillApi(apiUrl: string) {
+    await fillInput(rhvView.ovirtApiInput, apiUrl);
+  }
+
+  async fillUsername(username: string) {
+    await fillInput(rhvView.ovirtUsernameInput, username);
+  }
+
+  async fillPassword(password: string) {
+    await fillInput(rhvView.ovirtPasswordInput, password);
+  }
+
+  async fillCertificate(certificate: string) {
+    await fillInput(rhvView.ovirtCertInput, certificate);
+  }
+
+  async selectCluster(instanceConfig: rhvConfig) {
+    await selectOptionByText(rhvView.ovirtClusterSelect, instanceConfig.cluster);
+  }
+
+  async saveInstance(saveInstance: boolean) {
+    await setCheckboxState(rhvView.connectRhvInstanceButton, saveInstance);
+  }
+
+  async configureProvider(instanceConfig: rhvConfig) {
+    await this.fillApi(instanceConfig.apiUrl);
+    await this.fillCertificate(instanceConfig.certificate);
+    await this.fillUsername(instanceConfig.username);
+    await this.fillPassword(instanceConfig.password);
+    await this.saveInstance(instanceConfig.saveInstance);
+  }
+
+  async configureInstance(instanceConfig: InstanceConfig) {
+    // eslint-disable-next-line no-console
+    console.log('Entered configureInstance');
+    await selectOptionByText(rhvView.ovirtInstanceSelect, instanceConfig.instance);
+    // eslint-disable-next-line no-console
+    console.log('selectOptionByText performed');
+    if (instanceConfig.instance === IMPORT_WIZARD_CONN_TO_NEW_INSTANCE) {
+      // eslint-disable-next-line no-console
+      console.log('before configureProvider');
+      await this.configureProvider(instanceConfig);
+      // eslint-disable-next-line no-console
+      console.log('after configureProvider');
+    } else {
+      throw Error('Saved provider instances are not implemented');
+    }
+  }
+
+  async connectToInstance() {
+    await click(rhvView.connectRhvInstanceButton);
+  }
+
+  async selectSourceVirtualMachine(sourceVirtualMachine: string) {
+    await selectOptionByText(rhvView.ovirtVmSelect, sourceVirtualMachine);
+  }
+
+  async confirmAndCreate() {
+    await click(view.importButon);
+  }
+
+  // async navigateToDetail() {
+  //   await click(view.seeDetailPageButton);
+  //   await isLoaded();
+  // }
+
+  async waitForCreation() {
+    await browser.wait(
+      until.textToBePresentInElement(wizardView.creationSuccessResult, RHV_WIZARD_CREATE_SUCCESS),
+      PAGE_LOAD_TIMEOUT_SECS,
+    );
+  }
+
+  async import(config: VMImportConfig) {
+    const {
+      provider,
+      instanceConfig,
+      name,
+      description,
+      sourceVMName,
+      storageResources,
+      networkResources,
+    } = config;
+    const importWizard = new ImportWizard();
```

Comment:
we can use `this` instead of `new ImportWizard`

Reply:
Please elaborate

Reply:
fixed